### ISSUE TITLE
1

### ISSUE THREAD

**Issue by hzxhzxyaojiayou** [luo3300612/LaRE#11]

**Comment by hzxhzxyaojiayou** [comment]
> loss = F.mse_loss(model_pred.float(), target.float(), reduction="none") # bs * 4 * 32 * 32 这个是作者给出的在extract_lare.py中的mAIN中的代码。
> 
> 根据作者的论文，不应该是做VAE编码后的潜向量跟去噪后的潜向量的loss
